### ISSUE TITLE
Authorize fails on 500 error (probably instead of 422)

### ISSUE THREAD

**Issue by JakubMatejka** [keboola/oauth-api#72]
<img width="473" alt="1 docker-compose run --service-ports app docker 2018-08-19 21-37-34" src="https://user-images.githubusercontent.com/215660/44312424-21dabb80-a3f8-11e8-83f7-6b84aaddc2bd.png">

**Comment by MiroCillik** [comment]
Jo, uz to chapem, ono to totiz musis poslat ako `x-www-form-urlencoded`:
Aj ked tu je to tiez naspravne napisane ako `form-data` :/
https://oauthv2.docs.apiary.io/#reference/authorize/generate-oauth-token-for-oauth-applications/generate-token-from-a-web-form/ui

Kazdopadne tam dam nejaku podmienku na `Content-Type` alebo napada ta ako vynutit ten `Content-Type` napr. v `serverless.yml`?

**Comment by JakubMatejka** [comment]
Tak pořád to failuje. (Akorát moc nevím, proč se ta chyba nazalogovala správně do toho jsonu pro Papertrail)
<img width="551" alt="oauth-api 2018-08-31 10-47-25" src="https://user-images.githubusercontent.com/215660/44902999-b94ee100-ad0b-11e8-9867-d7bd52f46b66.png">

**Comment by MiroCillik** [comment]
Vyskusam teda tiez po deployi, 403 vyzera ako nejaky problem s pravami :/

**Comment by MiroCillik** [comment]
Aaha, uz to mam, totiz ta tvoja domena nie je pridana medzi callback url v nastaveniach Twitter apky :)
Upravim teda tu chybovu hlasku.
